src/OrbitQt/TimeGraphLayoutWidgetTest.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QApplication>
+
+#include "TimeGraphLayoutWidget.h"
+
+TEST(TimeGraphLayoutWidgetTest, SetScale) {
+  orbit_qt::TimeGraphLayoutWidget widget{};
+  // There is not a lot of logic in the TimeGraphLayoutWidget, but we can at least test the SetScale
+  // function.
+
+  const float previous_text_box_height = widget.GetTextBoxHeight();
```

Comment:
Nit: Consider to test all methods. I know it's a bit boring, but we can check that all the window scale as expected and we are not forgetting some element.

Reply:
ACK. :-)